modules/src/tiovx_aewb_module.c

```diff
@@ -386,6 +386,8 @@ vx_status tiovx_aewb_create_node(NodeObj *node)
                                 aewb_out,
                                 node_priv->dcc_config);
 
+    sprintf(node->name, "aewb_node");
```

Comment:
This should be done in init node
if you call it here then you are not giving user a chance to set the name

Reply:
Adressed and fixed.